packages/react/src/components/List/ListItem/_list-item.scss

```diff
@@ -207,7 +207,13 @@
         text-overflow: ellipsis;
         padding-right: $spacing-09;
         &__with-actions {
-          padding-right: $spacing-07; // save room for the action button
+          // save room for the action button
+          padding-right: $spacing-08;
```

Comment:
We should move away from the absolute positioned actions in the list that require special padding depending on the document direction. There are better ways of solving the layout, which I'm addressing in #3062

Reply:
Sure thing.